packages/commonwealth/server/router.ts

```diff
@@ -270,24 +270,28 @@ function setupRouter(
   router.post(
     '/createThread',
     passport.authenticate('jwt', { session: false }),
-    createThread.bind(this, models, tokenBalanceCache, ruleCache, banCache)
-  );
-  router.put(
-    '/editThread',
-    passport.authenticate('jwt', { session: false }),
-    editThread.bind(this, models, banCache)
+    [
+      databaseValidationService.validateAuthor.bind(databaseValidationService),
```

Comment:
Is there any way we can expose really clean function here that doesn't require this messy bind?

Reply:
refactored so we use arrow functions, so binding is not necessary!